app/src/main/java/com/example/nockanakalinowej/AnimationMenager.java

```diff
@@ -0,0 +1,86 @@
+package com.example.nockanakalinowej;
+
+import android.animation.Animator;
+import android.animation.AnimatorListenerAdapter;
+import android.animation.AnimatorSet;
+import android.animation.ObjectAnimator;
+import android.view.View;
+
+/**
+ * Animation Manager
+ */
+class AnimationMenager {
+    AnimatorSet animationSet;
+    Level window;
+
+    public AnimationMenager(Level context){
+        animationSet = new AnimatorSet();
+        window = context;
+    }
+
+    void startAnimation(final View tile1, final View tile2){
+        ObjectAnimator scaleXTile1 = ObjectAnimator.ofFloat(tile1, "scaleX", 1.2f);
+        scaleXTile1.setDuration(1000);
+        ObjectAnimator scaleYTile1 = ObjectAnimator.ofFloat(tile1, "scaleY", 1.2f);
+        scaleYTile1.setDuration(1000);
+        ObjectAnimator translationXTile1 = ObjectAnimator.ofFloat(tile1, "translationX", tile2.getX()-tile1.getX());
+        translationXTile1.setDuration(1000);
+        ObjectAnimator translationYTile1 = ObjectAnimator.ofFloat(tile2, "translationY", tile1.getY()-tile2.getY());
+        translationYTile1.setDuration(1000);
+        ObjectAnimator scaleXTile2 = ObjectAnimator.ofFloat(tile2, "scaleX", 1.2f);
+        scaleXTile2.setDuration(1000);
+        ObjectAnimator scaleYTile2 = ObjectAnimator.ofFloat(tile2, "scaleY", 1.2f);
+        scaleYTile2.setDuration(1000);
+        ObjectAnimator translationXTile2 = ObjectAnimator.ofFloat(tile2, "translationX", tile1.getX()-tile2.getX());
+        translationXTile2.setDuration(1000);
+        ObjectAnimator translationYTilel2 = ObjectAnimator.ofFloat(tile1, "translationY", tile2.getY()-tile1.getY());
+        translationYTilel2.setDuration(1000);
+        ObjectAnimator reScaleXTile1 = ObjectAnimator.ofFloat(tile1, "scaleX", 1f);
+        scaleXTile1.setDuration(1000);
+        ObjectAnimator reScaleYTile1 = ObjectAnimator.ofFloat(tile1, "scaleY", 1f);
+        scaleYTile1.setDuration(1000);
+        ObjectAnimator reScaleXTile2 = ObjectAnimator.ofFloat(tile2, "scaleX", 1f);
+        reScaleXTile2.setDuration(1000);
```

Comment:
Why did you change scaleXTile2 to reScaleXTile2? These are different objects!

Reply:
scaleXTile is set above, this is correct place for rescaling, however it should be change also in lines: 39 and 41